src/test/java/org/mule/service/scheduler/internal/config/ThreadPoolsConfigTestCase.java

```diff
@@ -365,15 +368,45 @@ public void missingValue() throws IOException, MuleException {
   @Description("Tests that the mule.schedulerPools.configFile property is honored if present")
   public void overrideConfigFile() throws IOException, MuleException {
     final Properties props = buildDefaultConfigProps();
-    props.put(IO_PREFIX + "." + THREAD_POOL_SIZE_MAX, "1");
+    props.put(IO_PREFIX + "." + THREAD_POOL_SIZE_MAX, "100");
 
     File overrideConfigFile = new File(tempOtherDir.getRoot(), "overriding.conf");
     props.store(new FileOutputStream(overrideConfigFile), "defaultConfig");
     System.setProperty(SCHEDULER_POOLS_CONFIG_FILE_PROPERTY, overrideConfigFile.getPath());
 
     final SchedulerPoolsConfig config = loadThreadPoolsConfig();
 
-    assertThat(config.getIoMaxPoolSize().getAsInt(), is(1));
+    assertThat(config.getIoMaxPoolSize().getAsInt(), is(100));
+  }
+
+  @Test
+  @Description("Tests that the mule.schedulerPools.configFile pointing to an external url property is honored if present")
+  public void overrideConfigFileWithUrl() throws IOException, MuleException {
+    final Properties props = buildDefaultConfigProps();
+    props.put(IO_PREFIX + "." + THREAD_POOL_SIZE_MAX, "1");
+
+    System.setProperty(SCHEDULER_POOLS_CONFIG_FILE_PROPERTY,
```

Comment:
Maybe I would upload in test/resources a copy of scheduler-pools.conf, in order to avoid possible failures due to future configuration changes